openpype/hosts/blender/api/utils.py

```diff
@@ -312,3 +314,38 @@ def transfer_stack(
             }
             for attr in attributes:
                 setattr(target_data, attr, getattr(stack_datablock, attr))
+
+
+def make_paths_absolute(source_filepath: Path = None):
+    """Make all paths absolute for datablock in current blend file.
+
+    Args:
+        source_filepath (Path, optional): Filepath to remap paths from,
+            in case file copy has been executed without paths remapping.
+            Defaults to None.
+    """
+    # In case no source filepath try naive system
+    if not source_filepath:
+        bpy.ops.file.make_paths_absolute()
```

Comment:
This will fail if this method is used on an unsaved blend file (i.e. before build first workfile). Then it might not be problematic in practice.